packages/esm-patient-forms-app/src/index.ts

```diff
@@ -117,11 +117,22 @@ function setupOpenMRS() {
         },
       },
       {
-        name: 'clinical-form-action-menu-item',
+        name: 'clinical-form-action-menu',
         slot: 'action-menu-items-slot',
         load: getAsyncLifecycle(() => import('./clinical-form-action-button.component'), options),
         order: 2,
       },
+      {
```

Comment:
Ok, one small ask here. We have a function in patient-common-lib called "registerWorkspace" that's supposed to replace the "workspace-as-extension" paradigm. It'd be good if we could use that here.

Reply:
We wouldn't want to use that here as it's currently implemented because that'd have the workspace interpreting this dashboard as a Clinical form. This means each time a user launches a form from the list, the workspace would bring up the `There is an active form open in the workspace` warning modal.